spec/integration/instance_properties_spec.cr

```diff
@@ -0,0 +1,144 @@
+require "./spec_helper"
+
+describe "C++ instance properties" do
+  it "works" do
+    build_and_run("instance_properties") do
+      # expose private methods as public ones
+      class MyProps < Test::Props
+        def x_prot
+          super
+        end
+
+        def x_prot=(x)
+          super
+        end
+
+        def y_prot
+          super
+        end
+      end
+
+      context "getter methods" do
+        it "is generated for public members" do
+          props = Test::Props.new(5, 8)
+          props.x_pub.should eq(5)
+          props.y_pub.should eq(8)
+        end
```

Comment:
Could also check that x_pub and y_pub are Crystal `public`, like the next two do